### ISSUE TITLE
Prevent flash of alt text in Firefox

### ISSUE THREAD

**Issue by muster-mark** [muster-mark/mark-fisher-photography#5]
Flash of orange alt text is jarring. Don't want to break alt text in case of images not loading.

Proposed solution. Have background color for images. Start with alt text as same colour, then animate to orange after ~2s. This should be enough time for the images to load, and if they don't then it will be only a short time until the alt text becomes visible.

**Comment by muster-mark** [comment]
Fixed in a8046f8a54ade63450f3838132017b75d7bbc6fc

**Comment by muster-mark** [comment]
Made alt text transparent instead of having background-color on img.
